qiskit/utils/parallel.py

```diff
@@ -151,13 +150,6 @@ def func(_):
     if len(values) == 1:
         return [task(values[0], *task_args, **task_kwargs)]
 
-    Publisher().publish("terra.parallel.start", len(values))
-    nfinished = [0]
-
-    def _callback(_):
-        nfinished[0] += 1
-        Publisher().publish("terra.parallel.done", nfinished[0])
-
```

Comment:
What was the point of this, historically?

Reply:
It was originally used to have a progress bar for things like `parallel_map`. Overall there was a goal to have a decoupled event bus that different components could subscribe to for async processing. But it never was really used for anything more than progress bars.

Reply:
Ah, good to know, thanks.  It was one of the major differences from the version that lived in QuTiP, but I'd never actually looked into it to find out what it was for.